crates/graphql/src/schema/util.rs

```diff
@@ -0,0 +1,30 @@
+use juniper::{ParseScalarResult, ParseScalarValue, Value};
```

Comment:
can this be in scalars module. Likely to add more. not a util per say.

Reply:
still think we should track this in the scalars directory.

Reply:
Ah, missed this comment lol.